src/crimson/osd/ops_executer.h

```diff
@@ -233,7 +233,7 @@ class OpsExecuter : public seastar::enable_lw_shared_from_this<OpsExecuter> {
   }
 
   template <class Func>
-  auto do_write_op(Func&& f, bool um);
+  auto do_write_op(Func&& f, bool um=true);
```

Comment:
> Less visual clutter this way.

yeah. how about making this more explicit: we could go further by defining something like:
```c++
enum class modified_by {
  user,
  sys,
};

template <class Func, modified_by m = modified_by::user>
auto do_write_op(Func&& f)
{
  // ...
  user_modify = (m == modified_by::user);
}

///
  case CEPH_OSD_OP_WATCH:
    return do_write_op<modified_by::sys>([this, &osd_op] (auto& backend, auto& os, auto& txn) {
      return do_op_watch(osd_op, os, txn);
    });
///
```

Reply:
will implement the proposed change in a separate PR.

Reply:
#47033